chain/ethereum/types.go

```diff
@@ -0,0 +1,7 @@
+package ethereum
+
+// Public Ethereum JSON RPC nodes
+var RPC = []string{
```

Comment:
are these just for tests?

Reply:
initially for tests but we could use them for other purposes such as backfill too